python/uds.py

```diff
@@ -478,11 +487,11 @@ def _isotp_rx_next(self, rx_data: bytes) -> None:
         if end >= self.tx_len:
           self.tx_done = True
         if self.debug:
-          print(f"ISO-TP: TX - consecutive frame - idx={self.tx_idx} done={self.tx_done}")
+          print(f"ISO-TP: TX - consecutive frame - {hex(self._can_client.tx_addr)} idx={self.tx_idx} done={self.tx_done}")
       elif rx_data[0] == 0x31:
         # wait (do nothing until next flow control message)
         if self.debug:
-          print("ISO-TP: TX - flow control wait")
+          print(f"ISO-TP: TX - {hex(self._can_client.tx_addr)} flow control wait")
```

Comment:
let's be consistent and always put the addr/data at the end (after text description)

Reply:
done